src/main/java/reposense/model/RepoConfiguration.java

```diff
@@ -16,6 +14,9 @@
 import reposense.system.LogsManager;
 import reposense.util.FileUtil;
 
+/**
+ * Represents configuration information from CSV config file for a single repository.
+ */
 public class RepoConfiguration {
```

Comment:
Can update `RepoConfiguration#equals(Object)` method as well to check for the `AuthorConfiguration`